src/Storage/Storage.Management/help/Enable-AzStorageBlobDeleteRetentionPolicy.md

```diff
@@ -52,7 +52,7 @@ This command enables delete retention policy for the Blob service, and set delet
 ## PARAMETERS
 
 ### -AllowPermanentDelete
-Allow deletion of the soft deleted blob versions and snapshots.
+Allow deletion of the soft deleted blob versions and snapshots. Permanent delete for soft deleted items does not work for hierarchical namespace enabled (HNS/ADLS Gen2) accounts currently.
```

Comment:
@Vignesh-Microsoft 

The limitation is not from Powershell, but from server side. 
If we add the limitation here, when server side change, the doc will not be accurate. 

If we really want to add the limitation, a better way might be add a link for the doc which mentioned the limitation here.

Reply:
@blueww , yes, I created PR's for those too -
https://github.com/Azure/azure-docs-rest-apis/pull/5518
https://github.com/Azure/azure-docs-rest-apis/pull/5519

Reply:
Thanks for the update!

However, I would suggest we don't add the server limitation here in Powershell doc. 
This will make the Powershell doc be more difficult to maitain, when server change.

If we really want to add the limitation, a better way might be add a link for the doc which mentioned the limitation here, instead of describing the limitation in Powershell doc.

Reply:
@Vignesh-Microsoft  This doc is generated automatically according to EnableAzureStorageBlobDeleteRetentionPolicy.cs. If you wanna change this doc, please change EnableAzureStorageBlobDeleteRetentionPolicy.cs.